tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -780,6 +780,11 @@ Then('user {string} should be listed as {string} in the collaborators list on th
   return assertCollaboratorslistContains('user', user, { role })
 })
 
+Then('the share with user {string} should have no expiration information', async function (user) {
```

Comment:
```suggestion
Then('the share with user {string} should have no expiration date set on the WebUI', async function (user) {
```

if you give the resource name also to this step here, then you will not need to use the steps 
```
    When the user picks the row of file "lorem.txt" in the webUI
    And the user switches to "collaborators" tab in details panel using the webUI
```
in the tests

Reply:
fixed